packages/cozy-doctypes/src/Document.js

```diff
@@ -328,6 +328,31 @@ class Document {
     }
     return result
   }
+
+  /**
+   * Fetch in one request a batch of documents by id.
+   * @param  {String[]} ids - Ids of documents to fetch
+   * @return {Promise} - Promise resolving to an array of documents, unfound document are filtered
```

Comment:
Sure? It seems that we return  an object?

Reply:
The error case returned an object yes. Fixed.